.github/ISSUE_TEMPLATE/---need-help-with-node-js-.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "⁉️ Need help with Node.js?"
+about: Please file an issue in our help repo.
+
+---
+
+If you have a question about Node.js that is not a bug report or feature
+request, please post it in https://github.com/nodejs/help!
```

Comment:
Maybe also add in this and the node.js org one that this will be closed if opened?